scrooge-generator/src/main/resources/androidgen/struct_inner.mustache

```diff
@@ -85,7 +85,7 @@ public {{#is_final}}final {{/is_final}}{{#in_class}}static {{/in_class}}class {{
     return new {{name}}(this);
   }
 
-  @Override
+  @java.lang.Override
```

Comment:
since we're trying to avoid collisions, let's use _root_.java.lang.Override for good measure.